oom-frontend/src/app/site/pages/auth/signin/signin.component.ts

```diff
@@ -1,5 +1,7 @@
+import { AuthService } from './../../../../core/auth/auth.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
import { AuthService } from "./../../../../core/auth/auth.service";
```